CHANGELOG.md

```diff
@@ -376,7 +376,7 @@ _Aug 22, 2022_
 
 A big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
 
-- ✨ @michaldudak synced the Material Icons set with the latest from Google (#33988).  
+- ✨ @michaldudak synced the Material Icons set with the latest from Google (#33988).
```

Comment:
This change is unrelated. Is there any specific reason for it?

Reply:
It's about the trailing space, I have VS Code configured to automatically remove them, e.g. of a value: https://github.com/mui/mui-x/pull/6242/files#r990690291.

Reply:
This shouldn't apply to markdown as trailing whitespace is meaningful there.

Reply:
I think that we can enable https://github.com/updownpress/markdown-lint/blob/master/rules/009-no-trailing-spaces.md next 😁

```diff
diff --git a/.markdownlint-cli2.cjs b/.markdownlint-cli2.cjs
index 78ba7deec3..0b423d3689 100644
--- a/.markdownlint-cli2.cjs
+++ b/.markdownlint-cli2.cjs
@@ -4,6 +4,7 @@ module.exports = {
   config: {
     default: true,
     MD004: false, // MD004/ul-style. Buggy
+    MD009: true, // MD009/no-trailing-spaces
     MD013: false, // MD013/line-length. Already handled by Prettier.
     MD014: false, // MD014/commands-show-output. It's OK.
     MD024: { siblings_only: true }, // MD024/no-duplicate-heading/no-duplicate-header
```

Reply:
I see a few other usages of line breaks in the docs. It sometimes feels better to break a line without introducing a new paragraph, so I'd be in slight favor of not introducing this new rule.

Reply:
I personally prefer the cleanliness of trailing spaces, but an HTML tag is acceptable as well.

Reply:
> cleanliness of trailing spaces

I have updated my previous comment, I think that the issue is with predictability.

Reply:
The markdown document ideally should be well readable both as plain text and rendered. The inclusion of an HTML tag (`<br />`) harms the reading flow. The `\` character could be a bit better in this context.

Reply:
I have done the change from `<br>` to `\`, it works https://github.com/mui/material-ui/blob/1108af61b61eeffdb0ee4b9149668ccde624a4a3/CHANGELOG.md#5102.

I think that for the markdown, it's more important to be able to predict how the source will render, as ultimately the rendered markdown will be read x10?, x100? more than the markdown source. So I think that `<br>` is clearer than `\`, but `\` is still a lot better than a double space, so deal 😁.